test/kpod_history.bats

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
```

Comment:
I wanted to hold off on these, since I thought they were causing issues in Ryan's previous request, get the separation merged, and then work through why the $STORAGE_OPTS was causing failures.